src/views/product-landing/components/cards/icon-dict.ts

```diff
@@ -1,30 +0,0 @@
-import { IconBox16 } from '@hashicorp/flight-icons/svg-react/box-16'
```

Comment:
For now I've removed components used on the previous stubbed version of this landing page, as they're no longer relevant. Sad to have to re-do a bit of work, but also exciting that we have new designs 🙌 🚂